storage/s3/storage.go

```diff
@@ -18,3 +25,24 @@ func SetToContext(c *gin.Context, s3client s3iface.S3API) {
 func GetFromContext(c context.Context) s3iface.S3API {
 	return c.Value(key).(s3iface.S3API)
 }
+
+// PutS3Object uploads a object to s3.
+func PutS3Object(c context.Context, input *s3.PutObjectInput) (*s3.PutObjectOutput, error) {
```

Comment:
@Dzalevski do we need to wrap this method

Reply:
Yes

Reply:
Why? It is not necessary.